src/storage/tests.rs

```diff
@@ -451,19 +467,34 @@ mod test {
         let mut w = World::new();
         w.register_with_id::<FlaggedCvec>(1);
         w.register_with_id::<FlaggedCvec>(2);
-        let mut s1: Storage<FlaggedCvec, _> = w.write_with_id(1);
-        let mut s2: Storage<FlaggedCvec, _> = w.write_with_id(2);
 
-        for i in 0..15 {
-            // Test insertion flagging
-            s1.insert(Entity::new(i, Generation::new(1)), i.into());
-            assert!(s1.open().1.flagged(Entity::new(i, Generation::new(1))));
+        let mut entities = Vec::new();
```

Comment:
Why are you doing this complicated loop here? Wouldn't a simple `.collect()` be sufficient? Or given that you only use these entities once, no allocation at all?